rs/tests/driver/src/driver/prometheus_vm.rs

```diff
@@ -161,8 +245,8 @@ if uname -a | grep -q Ubuntu; then
   # k8s
   chmod g+s /etc/prometheus
   cp -f /config/prometheus/prometheus.yml /etc/prometheus/prometheus.yml
-  cp -R /config/grafana/dashboards/IC /var/lib/grafana/dashboards/
-  chown -R grafana:grafana /var/lib/grafana/dashboards/IC/
+  cp -R /config/grafana/dashboards/k8s-dashboards /var/lib/grafana/dashboards/
```

Comment:
Is this `k8s-dashboards` sub directory needed? Wouldn't this cause different dashboard URLs as we have for manner?

Reply:
In k8s we have a `kustomization.yaml` which has `k8s-sidecar-target-directory` which contains the names of the directories. I will use it to fetch the correct folder names. Nice catch!